### ISSUE TITLE
ncRPI data

### ISSUE THREAD

**Issue by terry-r123** [LucaOne/LucaOne#32]
Thanks to the authors for their great work.

I would like to ask about the ncRPI task, how was the 16658 data in the training set and the 4166 data in the test set constructed? I can't match that number with those NPInter datasets from the original article ncRPI-LGAT: Prediction of ncRNA-protein interactions with line graph attention network framework.

**Comment by terry-r123** [comment]
> > Thanks to the authors for their great work.
> > I would like to ask about the ncRPI task, how was the 16658 data in the training set and the 4166 data in the test set constructed? I can't match that number with those NPInter datasets from the original article ncRPI-LGAT: Prediction of ncRNA-protein interactions with line graph attention network framework.
> 
> Please see the 5-fold dataset of ncRPI-LGAT: https://github.com/NWPU-903PR/ncRPI-LGAT/tree/master/data/set_allInteractionKey, NPInter2_lncRNA_0, NPInter2_lncRNA_1, NPInter2_lncRNA_2, NPInter2_lncRNA_3, NPInter2_lncRNA_4
> 
> If that helped you, please give LucaOne a star, tks.

Thanks for your reply. 
I was curious about the difference between NPInter2_lncRNA_0 and NPInter2_lncRNA_1, and the difference between set_interactionKey_train_0 and set_interactionKey_train_1. And it looks like the number of test sets in LucaOne is the same as in the set_negativeInteractionKey_all file?

**Comment by terry-r123** [comment]
> > > > Thanks to the authors for their great work.
> > > > I would like to ask about the ncRPI task, how was the 16658 data in the training set and the 4166 data in the test set constructed? I can't match that number with those NPInter datasets from the original article ncRPI-LGAT: Prediction of ncRNA-protein interactions with line graph attention network framework.
> > > 
> > > 
> > > Please see the 5-fold dataset of ncRPI-LGAT: https://github.com/NWPU-903PR/ncRPI-LGAT/tree/master/data/set_allInteractionKey, NPInter2_lncRNA_0, NPInter2_lncRNA_1, NPInter2_lncRNA_2, NPInter2_lncRNA_3, NPInter2_lncRNA_4
> > > If that helped you, please give LucaOne a star, tks.
> > 
> > 
> > Thanks for your reply. I was curious about the difference between NPInter2_lncRNA_0 and NPInter2_lncRNA_1, and the difference between set_interactionKey_train_0 and set_interactionKey_train_1. And it looks like the number of test sets in LucaOne is the same as in the set_negativeInteractionKey_all file?
> 
> Sorry, in the ncRPI dataset publicly available at LucaOne, the dataset is from https://github.com/NWPU-903PR/ncRPI-LGAT/tree/master/data/set_allInteractionKey/NPInter2.
> 
> set_interactionKey_train_0 ~ set_interactionKey_train_4 are positives for 5 times model training. set_negativeInteractionKey_train_0 ~ set_negativeInteractionKey_train_4 are negatives for 5 times model training.
> 
> Correspondingly, set_interactionKey_test_0 ~ set_interactionKey_test_1 are positives for 5 times model testing. set_negativeInteractionKey_test_0 ~ set_negativeInteractionKey_test_4 are negatives for 5 times model testing.
> 
> Using the five datasets for 5 times model training and testing:
> 
> set_interactionKey_train_0 + set_negativeInteractionKey_train_0 for the first model training, set_interactionKey_test_0 + set_negativeInteractionKey_test_0 for the first model testing.
> 
> ...
> 
> set_interactionKey_train_4 + set_negativeInteractionKey_train_4 for the 5th model training, set_interactionKey_test_4 + set_negativeInteractionKey_test_4 for the 5th model testing.
> 
> In the ncRPI dataset published by LucaOne, the training set(16,658 pairs) is from set_interactionKey_train_2 + set_negativeInteractionKey_train_2, and the testing set(2,083 pairs) is from set_interactionKey_test_2 + set_negativeInteractionKey_test_2.

Thanks for your detailed explanation. I have understood and given the star for the LucaOne.
